src/window.js

```diff
@@ -173,12 +176,21 @@ export default function DocumentationViewer({ application }) {
     button_forward.sensitive = webview.can_go_forward();
   }
 
-  button_back.connect("clicked", () => {
-    webview.go_back();
+  button_back.connect("clicked", () => { webview.go_back(); });
+  button_forward.connect("clicked", () => { webview.go_forward(); });
+
+  change_buttons_breakpoint.connect("apply", breakpoint => {
+    content_header_bar.remove(button_back);
+    content_header_bar.remove(button_forward);
+    bottom_toolbar.append(button_back);
+    bottom_toolbar.append(button_forward);
```

Comment:
Let's move this into a function ``moveButtonsDown()`` and call it here so that it's clear what's going on